bin/oneoff/back_populate_hoc_survey_prize_purpose

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env ruby
+
+# This script populates the purpose field in Pegasus.hoc_survey_prizes for claimed prizes with no purpose to "Hoc2014"
+
+require File.expand_path('../../../pegasus/src/env', __FILE__)
+
+DB[:hoc_survey_prizes].exclude(claimant: nil).where(purpose: '').update(purpose: 'Hoc2014')
```

Comment:
Those are unclaimed prizes in the pool. They don't have a purpose until they're [claimed](https://github.com/code-dot-org/code-dot-org/blob/66d3bb58af12e0c89a95329a7c63a7141c243c43/pegasus/helpers/prize_helpers.rb#L7-L7)